dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -159,11 +160,7 @@ void Trace::configureSink(const std::vector<TraceAppender>& appenders) {
   logging::attributes::current_thread_id::value_type currentId =
     logging::attributes::current_thread_id().get_value().extract<logging::attributes::current_thread_id::value_type>().get();
 
-  TraceSinks traceSink;
-  if (sinks_.find(currentId) != sinks_.end()) {
-    TraceSinks& traceSinkOld = sinks_.at(currentId);
-    traceSink.persistantSinks = traceSinkOld.persistantSinks;
-  }
+  TraceSinks& traceSink = sinks_[currentId];
```

Comment:
It means that it there was remaining non persistant sinks they are copied while it was not the case initially

Reply:
When Trace::configureSink() is called inside Trace::addAppenders(), Trace::resetCustomAppenders() is called previously so it's useless to clear non-persistant sinks a second time.

And when Trace::configureSink() is called inside Trace::clearLogFile(), we don't want to clear non-persistant sinks.

Reply:
This assumption is true today. Bu what happens if we write a new method that calls this? Better keep the safe way.